lib/features/family/features/reflect/presentation/pages/reveal_secret_word.dart

```diff
@@ -71,7 +71,10 @@ class _RevealSecretWordScreenState extends State<RevealSecretWordScreen> {
                   color: Colors.grey,
                   onChange: (value) => setState(() {
                     _isScratched = value > 25;
-                    text = 'Sneak your secret word\ninto ONE of your answers!';
+                    if (_isScratched) {
```

Comment:
Why would we change the text in setstates and base the text on _isScratched value? :) 
It's not really clear if the different texts are all over this file

Reply:
i changed the var names to make it clearer